conversant/demo/streamlit_example.py

```diff
@@ -290,9 +292,8 @@ def update_prompt_from_json() -> None:
             # The session's state needs to be manually updated since we are not
             # refreshing the entire Streamlit app.
             if not st.session_state.bot.chat_history:
-                st.session_state.bot.reply(
-                    query="Hello",
-                )
+                st.session_state.message_input = "Hello"
```

Comment:
This lets us use all the partial reply goodies for the first generated bot response